apps/rehabstod/src/pages/Patient/components/OpenTabsDialog.tsx

```diff
@@ -0,0 +1,56 @@
+import { IDSButton, IDSDialog, IDSDialogActions, IDSDialogElement } from '@frontend/ids-react-ts'
+import { useCallback, useEffect, useRef } from 'react'
+import { unstable_useBlocker as useBlocker } from 'react-router-dom'
+import { usePatient } from '../hooks/usePatient'
+
+export function OpenTabsDialog() {
```

Comment:
Should we add some tests for this new component?

Reply:
I created a new ticket for adding tests, I think it could become a bit tricky to test this component.